Documentation/doc/scripts/pkglist_filter.py

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env python3
+#!/usr/bin/env python
```

Comment:
Note that some systems do not have anything called `python`, they only have `python3` (and possibly `python2`). I don't know if the reverse exists.

Reply:
I did not think about that. Maybe I should replace with python3 but it does not really matter as the python exe used is always the one found at cmake level.

Reply:
This line could also just be removed, I think. Unless I'm wrong, it's only used if someone tries to launch the script as an executable using `./script.py`, while using `python script.py` will work without it.